ui/client/components/graph/EspNode/getRoundedRectPath.ts

```diff
@@ -0,0 +1,26 @@
+import {isArray} from "lodash"
+
+function toArray(r: number | number[]) {
```

Comment:
Please move it to arrayUtils and make number of elements (I mean this 10) as parameter.